app_dart/lib/src/service/github_service.dart

```diff
@@ -153,6 +153,20 @@ class GithubService {
     );
   }
 
+  /// Adds labels to an issue.
+  ///
+  /// A pull request is an issue. This works for pull requests as well.
+  Future<List<IssueLabel>> addIssueLabels(
+    RepositorySlug slug,
+    int issueNumber,
+    List<String> labels,
+  ) async {
+    ArgumentError.checkNotNull(slug);
```

Comment:
Does this throw? I am assuming this is similar to using assert?

Reply:
Yes. Exactly.